gui/js/sharad.js

```diff
@@ -0,0 +1,25 @@
+class SharadApi extends IApi {
+    name = "sharad";
+
+    /**
+     * @param file {string}
+     * @param time {string|number}
+     */
+    addSharad(file, time) {
```

Comment:
If you call this method just `add`, we could write `callJavascript("CosmoScout.sharad.add",...)` instead off `callJavascript("CosmoScout.call", "sharad", "addSharad",...)` which increases readability alot.

Reply:
Done in https://github.com/cosmoscout/csp-sharad/pull/3/commits/135e25828bc6a03460dd750247cbea6f24c86e66